vw_golf_mk4.dbc

```diff
@@ -321,7 +322,8 @@ BO_ 1160 Motor_6: 8 XXX
  SG_ Frei_Motor_6_4 : 58|2@1+ (1,0) [0|0] "" XXX
  SG_ ltemperaturschutz : 57|1@1+ (1,0) [0|0] "" XXX
  SG_ GRA_Bremseingriff_Freigabe : 56|1@1+ (1,0) [0|0] "" XXX
- SG_ Frei_Motor_6_3 : 40|16@1+ (1,0) [0|0] "" XXX
+ SG_ Frei_Motor_6_3 : 48|8@1+ (1,0) [0|0] "" XXX
+ SG_ Ruckmeldung_Momenten : 40|8@1+ (0.39,0) [0|100] "" XXX
```

Comment:
Same, revert.

Reply:
Please confirm if I should revert new field "Ruckmeldung_Momenten"